scenarios/csharp/dotnetcore/call-center/README.md

```diff
@@ -0,0 +1,46 @@
+# Scenarios: Post-call transcription and analytics with C# for .NET Core on Windows
+
+This sample demonstrates how to use post-call transcription and analytics with C# .NET 6.0 on Windows.
+
+## Run the Sample within VS Code
+- Install "Azure AI Speech Toolkit" extension in VS Code.
+- Download this sample from sample gallery to local machine.
+- Trigger "Azure AI Speech Toolkit: Configure Azure Speech Resources (AI Services)" command from command palette to select an Azure AI services resource.
```

Comment:
- Trigger "Azure AI Speech Toolkit: Configure Azure Speech Resources" command from command palette to select an __Azure AI services__ resource.

Reply:
Updated.